lib/juttle-viewer-view.coffee

```diff
@@ -0,0 +1,43 @@
+{ScrollView} = require 'atom-space-pen-views'
+JuttleClient = require 'juttle-client-library'
+
+module.exports =
+class JuttleViewerView extends ScrollView
+  @content: ->
+    @div class: 'juttle-viewer native-key-bindings', tabindex: -1
+
+  constructor: (@editorId) ->
+    super
+    juttleClient = new JuttleClient(atom.config.get('juttle-viewer.juttleServiceHost'))
+    @juttleClientView = new juttleClient.View(@element);
```

Comment:
You might want to do this with `juttle-viewer@0.7.0-rc.5` as the invocation changes.